src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -680,16 +682,45 @@ private void testSessionViewableTable() {
         assertTrue(feedbackPage.isDisabled(By.id("publishtime")));
     }
 
-    private void testDatePickerScripts() {
+    private void testDatePickerScripts(LocalDate defaultStartDate) {
```

Comment:
What's the rationale for passing the default start date as a parameter?

Reply:
ooops missed this comment...

The default start date and time of a new session is set to be the hour when the feedback session page is loaded.
So this `defaultStartDate` is created (line 656) when the feedback page for this test is created (line 655). This `defaultStartDate` is used to check against the max date of visible date in test `testDatePickerScripts`. 

However before the test `testDatePickerScripts`, the test `testSessionViewableTable` is run for some time. Hence, if the `defaultStartDate` is created in the test `testDatePickerScripts` instead of the moment the `feedbackPage` is created, `defaultStartDate` might be assigned a later time than the actual start date and time shown on the feedback page and gives wrong expected result.